rocketchat-core/src/main/java/com/rocketchat/core/RestImpl.java

```diff
@@ -163,6 +169,85 @@ public void onResponse(Call call, Response response) throws IOException {
         });
     }
 
+    void getRoomFiles(String roomId,
+                      BaseRoom.RoomType roomType,
+                      String offset,
+                      String sortBy,
+                      Sort sort,
+                      final RoomCallback.GetFilesCallback callback) {
+        checkNotNull(offset,"roomId == null");
+        checkNotNull(offset,"roomType == null");
```

Comment:
`checkNotNull(roomType,"roomType == null");`

Reply:
the good and old copy and paste and forget to edit

Reply:
Thanks for catching it @rafaelks and yeah what I was to said @luciofm